org.kde.skanlite.json

```diff
@@ -13,6 +13,7 @@
     ],
     "finish-args": [
         "--device=all",
+        "--env=SANE_CONFIG_DIR=/var/config/sane.d:",
```

Comment:
Wouldn't that mean that you need to be an admin to setup this config? I would expect unprivileged users to be able to setup the config on their own thus the need for the wrapper (https://github.com/flathub/org.kde.skanlite/issues/14#issuecomment-2310681109) as the path is dynamic (includes the home dir) and can not be set in the manifest here.

Reply:
Inside flatpak sandbox `/var/config` = `$XDG_CONFIG_HOME` (bind-mount). It's same path but static which is why we don't need a wrapper.

Reply:
Awesome, I didn't know that. Thanks